cmake/FindPythonModule.cmake

```diff
@@ -121,12 +121,19 @@ set( PythonModule_${module_NAME}_PATH
   endif()
 
   include(FindPackageHandleStandardArgs)
-  set(FPHSA_NAME_MISMATCHED TRUE)
+  set(CMAKE_FIND_PACKAGE_NAME PythonModule_${module_NAME})
+  foreach(VAR IN ITEMS REQUIRED QUIETLY VERSION COMPONENTS)
```

Comment:
There are other variables as well. Whenever I need to do something like this, I refer to [this list](https://cmake.org/cmake/help/latest/command/find_package.html#id6).

Reply:
I went through the list and the individual components are handled differently so those are intentionally skipped.  The remaining are specific version components which should be okay just handling the whole version variable, right?

Reply:
If it works as is then I guess that's OK.